emr-user-role-mapper-application/src/main/java/com/amazon/aws/emr/ApplicationConfiguration.java

```diff
@@ -101,6 +101,12 @@ public void setProperty(String propertyName, String value) {
         return this.IMPERSONATION_ALLOWED_USERS;
     }
 
+    public boolean isRegionalStsEnabled() {
+        return Boolean.parseBoolean(properties.getProperty(Constants.REGIONAL_STS_ENABLED,
+            String.valueOf("true")));
+    }
+
+
```

Comment:
1/ Add some comments to explain this config
2/ nit: regionalStsEnabled -> regionalStsEndpointEnabled

Reply:
Done